docs/release/release-cycle.md

```diff
@@ -32,16 +29,23 @@ A release cycle can be split up into the following phases:
     * `x.y.0` GA release is created based on the release branch
 * After that:
     * **Note** The following is the responsibility of the release team of the following release cycle.
-    * `x.y.1+`: Monthly patch releases will be released based on the release branch
-    * Cherry-picks to the release branch are allowed according to the [backport policy](https://github.com/kubernetes-sigs/cluster-api/blob/main/CONTRIBUTING.md#backporting-a-patch)
-    * Providers create releases using the new CAPI minor release when they are ready
-    * Development of the next release can now officially start on the main branch
+    * `x.y.1+`-Monthly : Monthly patch releases will be released based on the release branch
+    * `x.y.1+`-Extra: In order to quickly provide official support for  a new Kubernetes minor release to Cluster API users,
```

Comment:
Is this proposal looking to add new tags with the name `x.y.z-monthly` and `x.y.z-extra`?

Reply:
Nope. We use the regular version format

Reply:
Changed to avoid confusion, PTAL